WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -81,6 +80,46 @@ final class PointOfSaleDashboardViewModelTests: XCTestCase {
         XCTAssertEqual(sut.isCartCollapsed, expectedCartCollapsedState)
     }
 
+    func test_isAddMoreDisabled_is_true_when_order_is_syncing_and_payment_state_is_idle() {
+        var sut: any PointOfSaleDashboardViewModelProtocol
+
+        sut = PointOfSaleDashboardViewModel(itemProvider: itemProvider,
```

Comment:
As explained in https://github.com/woocommerce/woocommerce-ios/pull/13310#discussion_r1680493636, we should just use `PointOfSaleDashboardViewModel` within `PointOfSaleDashboardViewModelTests`. However, all the dependencies that `PointOfSaleDashboardViewModel` depend on, can be mocked and used here.